### PR TITLE
fix REQUEST_URI for requests through FastCGI

### PR DESCRIPTION
**Pull request by marliotto**
For https://github.com/spiral/roadrunner/issues/232 issue

### CODE REVIEW

File: service/http/request.go [hunk 1]
```diff
@@ -170,6 +170,9 @@ func (r *Request) contentType() int {
 
 // uri fetches full uri from request in a form of string (including https scheme if TLS connection is enabled).
 func uri(r *http.Request) string {
+	if r.URL.Host != "" {
```
Comment (wolfy-j) [inline service/http/request.go]: So, in this case, we are fully relying on URL provided by parent service?

Comment (marliotto) [reply service/http/request.go]: Yes, I've investigated it. FastCGI returns full URL

**Comment by marliotto** [comment]
@wolfy-j, Could you help? 
How to fix "listen tcp 0.0.0.0:6082: bind: address already in use" in test? 
May be use new port?

**Comment by wolfy-j** [comment]
Yes, you have to rotate ports for RPC and your server. The issue is deep in Golang and Docker, sometimes it does not release port **immediately** withing the same process.

You can check the commits I did with text "stable tests", also see https://github.com/spiral/roadrunner/issues/220

**Comment by wolfy-j** [comment]
Thank you!

**Pull request merged**